react/components/Dropzone/README.md

```diff
@@ -50,6 +50,60 @@ class MyDropZone extends React.Component {
 ;<MyDropZone />
 ```
 
+Size and file type restriction
+
+```js
+class MyDropZone extends React.Component {
+  constructor() {
+    super()
+    this.state = { files: null, result: null }
+    this.handleFile = this.handleFile.bind(this)
+  }
+
+  handleFile(files) {
+    this.setState({ result: files })
+  }
+
+  handleReset(files) {
+    if (files) {
+      console.log(files)
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <Dropzone
+          accept=".xml"
```

Comment:
Could you choose another file extension? I know that is your case, but I think would be easier for everyone to test if it was a more common extension (as .jpg, .pdf, etc)

Reply:
The code can be edited live to on the page to use any extension or mime type you want.

Reply:
> Your _request changes_ are regarding this example not using the `onDropRejected` function with (say) an alert() inside or a more informative error for the component?

Exactly! Didn't know that's a limitation of the lib =( Is there any possible workaround for this? I think it would be a bad experience for users if it's not addressed.